test/index.test.js

```diff
@@ -64,12 +66,6 @@ describe('Outbound', function() {
     });
   });
 
-  describe('loading', function() {
-    it('should load', function(done) {
-      analytics.load(outbound, done);
-    });
-  });
```

Comment:
we like to leave it in to ensure in testing that the integration does load as expected. it's not a dealbreaker, I just wondered if you removed it for any particular reason? like if that test was failing etc